src/Mono.Android/Mono.Android.csproj

```diff
@@ -22,6 +22,7 @@
     <Nullable>enable</Nullable>
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
     <EnableSingleFileAnalyzer>true</EnableSingleFileAnalyzer>
+    <EnablePreviewFeatures>True</EnablePreviewFeatures>
```

Comment:
Probably won't be able to keep this line when we merge.

Reply:
We don't need it now, as I've updated Java.Interop to stop using the "preview features" attributes.  I just forgot about this.